specs/order_spec.rb

```diff
@@ -141,21 +170,49 @@
     end
 
     it "Returns accurate information about the last order" do
-      # TODO: Your test code here!
+      last = Order.all.last
+      expect(last.id).must_equal 100
     end
   end
 
   describe "Order.find" do
     it "Can find the first order from the CSV" do
-      # TODO: Your test code here!
+      first = Order.find(1)
+      expect(first).must_be_kind_of Order
+      expect(first.id).must_equal 1
     end
 
     it "Can find the last order from the CSV" do
-      # TODO: Your test code here!
+      last = Order.find(100)
+
+      expect(last).must_be_kind_of Order
+      expect(last.id).must_equal 100
+    end
+
+    it "Returns nil for an order that doesn't exist" do
+      expect(Order.find(53145)).must_be_nil
```

Comment:
It might be nice to give a name to the value `4500`, like `non_extant_order_id = 4500`